content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,60 +2,114 @@
-   navigate to :menuselection:`Inventory --> Configuration --> Settings`, scroll down to the
-   :guilabel:`Warehouse` heading, and enable the :guilabel:`Storage Locations` checkbox.
+In Odoo *Inventory*, a *location* is a specific space within a warehouse. This can be a shelf, room,
+aisle, rack, bin, etc.
+
+Locations allow for the storage of products in designated areas of the warehouse, to help track
+inventory accurately.
+
+Configuration
+=============
+
+In order to use locations, the :guilabel:`Storage Locations` setting must be enabled. To do so,
+navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and scroll down to the
+:guilabel:`Warehouse` section. Tick the checkbox next to :guilabel:`Storage Locations` to enable the
+feature, and click :guilabel:`Save`.
+
+.. image:: use_locations/use-locations-enabled-setting.png
+   :align: center
+   :alt: Enabled Storage Location setting in Inventory app settings.
+
+Once the :guilabel:`Storage Locations` feature is enabled, two additional internal link options
+appear below it: :guilabel:`Locations` and :guilabel:`Putaway Rules`. Click :guilabel:`Locations` to
+open a list of all existing locations in the warehouse.
 
 Create a new location inside a warehouse
 ========================================
 
-Starting from the :menuselection:`Inventory` app, select :menuselection:`Configuration --> Locations
---> Create`. The new location form can then be configured as follows:
+To create a new location inside a warehouse, navigate to :menuselection:`Inventory app -->
+Configuration --> Locations`. Doing so opens a list of all existing locations. Click :guilabel:`New`
+to open a new location creation form.
+
+In the :guilabel:`Location Name` field, assign a name for the new location. In the :guilabel:`Parent
+Location` field, assign the location that this new location exists in.
+
+.. example::
+   A new location `Gate 1` is created. Its parent location is `Dispatch Zone`. In the
+   :guilabel:`Parent Location` for `Gate 1`, select `Dispatch Zone`.
+
+In the :guilabel:`Additional Information` section, fill out the necessary details for the new
+location.
+
+In the :guilabel:`Location Type` field, select which type of location this is. There are seven
+different types of locations to choose from:
 
-- :guilabel:`Location Name`: the name that will be used to reference the location
-- :guilabel:`Parent Location`: the location or warehouse that the new location exists within
-- :guilabel:`Location Type`: choose the category that the location belongs to
-- :guilabel:`Company`: the company that owns the warehouse that the location is inside of
+- :guilabel:`Vendor Location`: Virtual location representing the source location for products coming
+  from vendors.
+- :guilabel:`View`: Virtual location used to create a hierarchical structure for a warehouse;
+  aggregates its child locations. **Cannot directly contain products**.
```

Comment:
> Cannot directly contain products

It can, though it shouldn't, so let's reword that (:

```suggestion
  aggregates its child locations. **Should not directly contain products**.
```

Reply:
Suggestion implemented ✅